spec/requests/v6/assessments_controller/realistic_civil_apply_spec.rb

```diff
@@ -0,0 +1,244 @@
+require "rails_helper"
+
+# Purpose of this spec is to test the real submission by the "Civil Apply" covering as many fields filled in as possible
+# Civil apply never send data for employment_details, self_employment_details and partner
+
+module V6
+  RSpec.describe AssessmentsController, :calls_bank_holiday, type: :request do
+    describe "POST /create" do
+      before do
+        create :state_benefit_type, label: "personal_independent_payments"
+        post v6_assessments_path, params: params.to_json, headers:
+      end
+
+      let(:headers) { { "CONTENT_TYPE" => "application/json", "Accept" => "application/json", 'HTTP_USER_AGENT': "CivilApply/1.0 production" } }
+      let(:params) do
+        {
+          "assessment": {
+            "level_of_help": "certificated",
+            "submission_date": "2023-08-15",
+            "client_reference_id": "3000-01-01",
+          },
+          "applicant": {
+            "employed": true,
+            "date_of_birth": "1982-08-16",
+            "involvement_type": "applicant",
+            "has_partner_opponent": false,
+            "receives_qualifying_benefit": true,
+          },
+          "proceeding_types": [
+            {
+              "ccms_code": "DA004",
+              "client_involvement_type": "A",
+            },
+          ],
+          "capitals": {
+            "bank_accounts": [
+              {
+                "value": "520.8",
+                "description": "Current accounts",
+              },
+            ],
+            "non_liquid_capital": [
+              {
+                "value": "1000.0",
+                "description": "Any valuable items worth more than £500",
+              },
+            ],
+          },
+          "cash_transactions": {
+            "income": [
+              {
+                "category": "friends_or_family",
+                "payments": [
+                  {
+                    "date": "2023-04-01",
+                    "amount": 250.0,
+                    "client_id": "3000-01-01",
```

Comment:
again, I don't know the real values, but let's include some letters, rather than looking like a date